twine/package.py

```diff
@@ -65,6 +65,28 @@ def _safe_name(name: str) -> str:
     return re.sub("[^A-Za-z0-9.]+", "-", name)
 
 
+def _dedent(string: str) -> str:
+    """Remove line continuation suffix used for the ``Description`` metadata field.
+
+    The metadata standard prescribes that continuation lines should be
+    prefixed with 7 spaces followed by a pipe character, however, setuptools
+    used to prefix continuation lines with 8 spaces. Handle both here.
+
+    If not all lines following the first start with either of the accepted
+    prefixes, the string is returned without modifications.
+    """
+    lines = string.splitlines()
+    if (
+        False  # This is here only to force black into something sensible.
+        or all(line.startswith("       |") for line in lines[1:])
+        or all(line.startswith("        ") for line in lines[1:])
```

Comment:
```suggestion
        or all(line.startswith(("       |", "        ")) for line in lines[1:])
```

Reply:
This would implement a different check. The code as written tests whether all the line start with 8 spacer or all the lines start with 7 spaced and a pipe. The proposed code tests with all the lines start with either 8 spaces or 7 spaces and a pipe. I don't think we should treat intermixed continuation style as valid. There is a test that ensures that someone else is not tempted into the same erroneous simplification.